### PR TITLE
Bump recommended kops versions in alpha channel

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: channels/alpha [hunk 1]
```diff
@@ -86,20 +86,24 @@ spec:
     recommendedVersion: 1.11.10
     requiredVersion: 1.11.10
   kopsVersions:
+  - range: ">=1.19.0-alpha.1"
+    #recommendedVersion: "1.19.0-alpha.1"
+    #requiredVersion: 1.19.0
+    kubernetesVersion: 1.18.3
```
Comment (rifelpet) [inline channels/alpha:92]: Can you explain why this is 1.18.3? should we not k8s beta releases here? I see that kops 1.17 used 1.17.0-beta.2 here:

https://github.com/kubernetes/kops/commit/6559a0e0317d301a1aea6d8745fed694901326df#diff-37c59376fa8938637370b21d9540af69R72-R76

Comment (johngmyers) [reply channels/alpha]: Okay, changed.

**Comment by rifelpet** [comment]
`curl: (56) OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 104` 🤔 

/retest

**Pull request merged**